tests/integration/test_pip_packages.py

```diff
@@ -0,0 +1,116 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from os import path, listdir, walk
+from requests import get
```

Comment:
I'd rather import just requests here and call requests.get when necessary, so we are explicit that we are using requests on get calls.